skbio/stats/composition.py

```diff
@@ -2327,6 +2341,7 @@ def dirmult_lme(
     --------
     statsmodels.formula.api.mixedlm
     statsmodels.regression.mixed_linear_model.MixedLM
+    statsmodels.stats.multitest.multipletests
     differential.regression.mixedlm
```

Comment:
Should we remove this?
```suggestion
```

Also, shouldn't we add `skbio.stats.composition.dirmult_ttest`?

Reply:
```suggestion
    differential.regression.mixedlm

  References
    ----------
    .. [1] Fernandes et al. "Unifying the analysis of
       high-throughput sequencing datasets: characterizing RNA-seq,
       16S rRNA gene sequencing and selective growth experiments by
       compositional data analysis." Microbiome (2014).
```
I'm also wondering if we should add in the aldex2 reference for the Dirichlet multinomial reference